src/modules/reportes_poai/reportes_poai.service.ts

```
@@ -0,0 +1,665 @@
+
+      // Crear datos del reporte manualmente
+      const data: any[] = [];
+      let rowNumber = 1;
+
+      // Procesar programación financiera
+      if (datosPoai?.programacion_financiera) {
+        for (const pf of datosPoai.programacion_financiera) {
+          // Obtener meta de producto
+          const { data: metaData } = await this.supabaseService.clientAdmin
+            .from('meta_producto')
+            .select(`
```

ESLint

[error] Insert `⏎··············`

(prettier/prettier)